lumigator/backend/backend/models.yaml

```
@@ -78,11 +78,11 @@
     - summarization:
     - translation:
 
-- display_name: open-mistral-7b
-  model: open-mistral-7b
+- display_name: ministral-8b-latest
```

```suggestion
- display_name: ministral-8b
```

Should we just go with `ministral-8b` for the display name? For example, with DeepSeek the model is called `deepseek-chat` but the display name is `deepseek-v3`.